README.md

```diff
@@ -1 +1,17 @@
 # hw01
+
+# My name is **Rachel Sass**. Here are a few things about me!
```

Comment:
This bolding won't render because of the heading (the heading takes precedence).